### PR TITLE
[TS] LPP46502 > LPS-159504 The height of the embedded documents and media widget (iframe) is not adapted when the view mode is changed to a shorter one

### PR DESCRIPTION
**Pull request by locpham97**
# Motivation

Dear Team,
Our customer found an issue that the height of the embedded documents and media widget (iframe) is not adapted when the view mode is changed to a shorter one
This PR is to solve that problem.
# Proposed Solution

Check if the content height is lower than default normal height, then set it to this value

# Steps to Verify

see https://issues.liferay.com/browse/LPS-159504 and https://issues.liferay.com/browse/LPS-159504

# References to existing code (if applies)

https://github.com/liferay-core-infra/liferay-portal/blob/master/modules/apps/iframe/iframe-web/src/main/java/com/liferay/iframe/web/internal/configuration/IFramePortletInstanceConfiguration.java line 84

### CODE REVIEW

File: modules/apps/layout/layout-type-controller/layout-type-controller-embedded/src/main/resources/META-INF/resources/layout/view/embedded_js.jspf [hunk 1]
```diff
@@ -22,17 +22,21 @@
 
 		iframe.on('load', () => {
 			var height = A.Plugin.AutosizeIframe.getContentHeight(iframe);
+			var heightNormal = 600;
```
Comment (diegonvs) [inline modules/apps/layout/layout-type-controller/layout-type-controller-embedded/src/main/resources/META-INF/resources/layout/view/embedded_js.jspf:25]: Hey @locpham97, what about renaming this variable to defaultHeight?

Comment (chileces) [reply modules/apps/layout/layout-type-controller/layout-type-controller-embedded/src/main/resources/META-INF/resources/layout/view/embedded_js.jspf:25]: hello @diegonvs , it is referred to as the existing variable in this file https://github.com/liferay-core-infra/liferay-portal/blob/master/modules/apps/iframe/iframe-web/src/main/java/com/liferay/iframe/web/internal/configuration/IFramePortletInstanceConfiguration.java line 84.

Comment (diegonvs) [reply modules/apps/layout/layout-type-controller/layout-type-controller-embedded/src/main/resources/META-INF/resources/layout/view/embedded_js.jspf:25]: Oh ok. I got it.

**Review by diegonvs — APPROVED** [review]

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by diegonvs** [comment]
ci:forward

**Pull request closed without merging**